__tests__/all_spec.js

```diff
@@ -29,7 +29,7 @@ names.forEach((name) => {
     let delays;
 
     beforeAll(async () => {
-      await page.goto(`http://localhost:${port}/${name}`);
+      await page.goto(`http://localhost:${port}/${name}/index.html`);
```

Comment:
Just curios, if you have any problem with the previous one?

Reply:
![redirect loop](https://user-images.githubusercontent.com/20126259/68626673-2c411380-04dc-11ea-9a6c-75e35e52259d.PNG)

maybe a bug of http-server and windows?

Reply:
gotcha.